### ISSUE TITLE
README

### ISSUE THREAD

**Issue by ghost** [mathiask88/node-snap7-testsuite#2]
Hi

Can you update the README file to know how to use this test suite ?

**Comment by mathiask88** [comment]
Yeah, sure I can do this in the next days. You just need to `clone->npm install->node app.js->http://127.0.0.1:8080`. This testsuite is not finished yet, but has some features to test :)

**Comment by ghost** [comment]
Thank You I figured it out :)
